seqr/views/apis/anvil_workspace_api_tests.py

```diff
@@ -469,16 +531,25 @@ def test_get_anvil_vcf_list(self, mock_subprocess, mock_file_logger, mock_utils_
         self.assertEqual(response.status_code, 200)
         self.assertDictEqual(response.json(), {'dataPathList': ['/test.vcf', '/data/test.vcf.gz']})
         mock_subprocess.assert_has_calls([
-            mock.call('gsutil ls gs://test_bucket', stdout=-1, stderr=-2, shell=True),
+            mock.call('gsutil -q ls gs://test_bucket', stdout=-1, stderr=-2, shell=True),
             mock.call().wait(),
-            mock.call('gsutil ls gs://test_bucket/**', stdout=-1, stderr=-2, shell=True),
+            mock.call('gsutil -q ls gs://test_bucket/**', stdout=-1, stderr=-2, shell=True),
             mock.call().wait(),
         ])
         mock_file_logger.info.assert_has_calls([
-            mock.call('==> gsutil ls gs://test_bucket', self.manager_user),
-            mock.call('==> gsutil ls gs://test_bucket/**', self.manager_user),
+            mock.call('==> gsutil -q ls gs://test_bucket', self.manager_user),
+            mock.call('==> gsutil -q ls gs://test_bucket/**', self.manager_user),
         ])
 
+        # Test merging sharded VCF files
+        mock_subprocess.return_value.stdout = [
```

Comment:
why is this a separate test case, why not just modify the above test to have both sharded and non-sharded VCFs?

Reply:
Good point.